build/scripts/unzip.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Unzip reports a warning that zip contains backslashes and returns exit code 1 
+# even though everything is good. Ignore exit code 1.
```

Comment:
@agocke have you seen this before?  I feel like we've worked through this issue.